Source/Core/Plugins/EventProcessor/EventContext.cs

```diff
@@ -3,15 +3,18 @@
 using Exceptionless.Core.Pipeline;
 using Exceptionless.Core.Utility;
 using Exceptionless.Core.Models;
+using Exceptionless.Core.Queues.Models;
 
 namespace Exceptionless.Core.Plugins.EventProcessor {
     public class EventContext : ExtensibleObject, IPipelineContext {
-        public EventContext(PersistentEvent ev) {
+        public EventContext(PersistentEvent ev, EventPostInfo epi = null) {
```

Comment:
Kinda stinks having our event context dependent upon the post info but I could see you wanting to do stuff with it..